src/content/docs/iast/troubleshoot-iast.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Troubleshooting for interactive applications security testing (IAST)
```

Comment:
This doc feels weird as a standalone thing. It doesn't quite much our troubleshooting doc guidelines: https://docs.newrelic.com/docs/style-guide/writing-docs/article-templates/troubleshooting-docs-guide/

I think I'd prefer to either stick this at the end of the Use IAST doc (with each problem statement in a series of collapsers for easy scanning) or have each of these split out into its own troubleshooting doc in a specific Troubleshooting category.

Reply:
I'll add it to the main doc, then!